packages/widget/src/stores/routes/utils.ts

```diff
@@ -6,20 +6,14 @@ export const isRouteDone = (route: RouteExtended) => {
 };
 
 export const isRoutePartiallyDone = (route: RouteExtended) => {
-  return route.steps.some(
-    (step) =>
-      step.execution?.process.some(
-        (process) => process.substatus === 'PARTIAL',
-      ),
+  return route.steps.some((step) =>
+    step.execution?.process.some((process) => process.substatus === 'PARTIAL'),
```

Comment:
Do you have a different code formatting tool? We should use the same prettier setup and not change existing code formatting 🙂

Reply:
I'll check but we should be using the same prettierrc file